pkg/vm/vm.go

```diff
@@ -956,7 +967,7 @@ func (v *VM) execute(ctx *Context, op Instruction, parameter []byte) {
 
 	case CALL:
 		v.istack.PushVal(ctx.Copy())
-		v.execute(v.Context(), JMP, parameter)
+		_ = v.execute(v.Context(), JMP, parameter)
```

Comment:
Which probably is wrong, if there was an error in inner context this should also fail, IMO.